system/Config/DotEnv.php

```diff
@@ -94,7 +94,7 @@ public function parse(): ?array
      */
     protected function setVariable(string $name, string $value = '')
     {
-        if (! getenv($name, true)) {
+        if (in_array(getenv($name, true), ['', '0', [], false], true)) {
```

Comment:
i think this is just checking if the env exists so we can just check `!== false`